NitroxClient/Communication/Abstract/IMultiplayerSessionState.cs

```diff
@@ -7,6 +7,7 @@ public interface IMultiplayerSessionState
     {
         IClient Client { get; }
         string IpAddress { get; }
+        int serverPort { get; }
```

Comment:
Could you make `serverPort` pascal case as `ServerPort`? Only for this property. We use pascal case for properties.